lib/new-admin/config/accounts.js

```diff
@@ -1,6 +1,5 @@
 const _ = require('lodash/fp')
 const { COINS, ALL_CRYPTOS } = require('./coins')
-const _ = require('lodash/fp')
```

Comment:
Something looks wrong here. I don't see why this is in the commit given that this line doesn't exist in dev.